query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/persist/CreateLdapConfiguration.java

```diff
@@ -145,6 +160,24 @@ public void validate() {
     if (containsErrorMsgs()) {
       return;
     }
+
+    List<StringField> claimArgs =
+        claimMappings
+            .getList()
+            .stream()
+            .map(ClaimsMapEntry::claimField)
+            .collect(Collectors.toList());
+
+    addErrorMessages(
+        SecurityValidation.validateStsClaimsExist(
+            claimArgs, configuratorSuite.getServiceActions(), stsServiceProperties));
+
+    claimMappings
```

Comment:
This validation is already executed above when super.validation() is called. It's part of the field's default validation https://github.com/connexta/admin-console/blob/17433eefd7d08af2e1a42bb6130411099d421a6b/query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/LdapAttributeName.java#L52